Main.java

```diff
@@ -9,7 +10,22 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
-  }
-}
+    // TODO code application logic here
+    Scanner input = new Scanner(System.in);
+    System.out.println ("Please enter an integer");
+    int first_Number = input.nextInt();
+    System.out.println("Please enter a second integer");
+    int second_Number = input.nextInt();
+
+    if (first_Number > second_Number) {
+      System.out.println (+ first_Number + " is bigger than " + second_Number + ".");
+    }
+      else if (first_Number < second_Number) {
+        System.out.println (+ second_Number + " is bigger than " + first_Number + "." );
+      }
+        else if (first_Number == second_Number) {
+          System.out.println (+ first_Number + " is equal to " + second_Number + ".");
+        }
+
```

Comment:
Watch the indentation here. The else if statements should be at the same indentation level as the if statement is. Also, the code that is inside the if statement should be at the same indented level as well.
Make sure you are trying to match the sample output as closely as you can so it will pass the test case.